File: Module.Frontend.TPM/Client/model/tpm/pludictionary/HistoricalPLUDictionary.js
Ext.define('App.model.tpm.pludictionary.HistoricalPLUDictionary', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'PLUDictionary',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'ClientTreeId', hidden: true, isDefault: true },
        { name: 'ClientTreeName', type: 'string', hidden: false, isDefault: true },
        { name: 'EAN_PC', type: 'string', hidden: false, isDefault: true },
        { name: 'PluCode', type: 'string', hidden: false, isDefault: true },
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalPLUDictionaries',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/History/HistoricalPLUDictionary.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;
using System;

namespace Module.Persist.TPM.Model.History
{
	[AssociatedWith(typeof(PLUDictionary))]
	public class HistoricalPLUDictionary : BaseHistoricalEntity<System.Guid>
	{
		public int ClientTreeId { get; set; }

		public string EAN_PC { get; set; }

		public string PluCode { get; set; }

		public string ClientTreeName { get; set; }

		public int ObjectId { get; set; }
	}
}
